util/test/sub_test

```diff
@@ -1311,6 +1318,14 @@ for testname in testsrc:
         print('[Elapsed compilation time for "{0}" - {1:.3f} '
             'seconds]'.format(test_name, elapsedCompTime))
 
+        # remove some_file: output from C compilers
+        if is_c_test:
+          for arg in args:
+            if arg.endswith(".c"):
+              # remove lines like
+              # somefile.c:
+              # that some C compilers emit when compiling multiple files
+              output = output.replace(arg + ":\n", "");
```

Comment:
Is there a way to stifle these messages in the compiler? This seems like a potentially fragile bit of code, but I'm ok with it if there isn't a way to stifle the messages.